docs/source/languages/orquesta.rst

```diff
@@ -101,6 +101,17 @@ transition will be represented as an outbound edge of the current task. When the
 specified in ``when`` of the transition is met, the next set of tasks under ``do`` will be invoked.
 If there are no more outbound edges identified, then the workflow execution is complete.
 
+Orquesta workflows are designed to fail fast. If a task fails which has no transition for that
```

Comment:
Please change the paragraph to the following.

```
The orquesta workflow engine is designed to fail fast and terminate the execution
of the workflow when a task fails and no remediation for the failed task is defined.
If the failed task has no transition for that failure condition, then the workflow will
stop as soon as any in-progress tasks are completed. To be more specific, if there
are multiple parallel branches and a task failed in one branch with no transition
defined for the failure, then the workflow engine will wait for all currently running
tasks in the other branches to finish and then terminate the workflow. No other tasks
from any branches will be queued or scheduled after the workflow is terminated. This
design allows users to quickly identify and address problems during runtime without
waiting for the other tasks in other branches to complete. The users can quickly fix the
problem and either rerun the workflow from the beginning or from where the workflow
failed.
```

Reply:
@m4dcoder many thanks for review - changes made.